frontend/src/metabase/query_builder/components/DatasetEditor/ResizableNotebook/ResizableNotebook.jsx

```diff
@@ -30,9 +67,9 @@ function ResizableNotebook({
         onResizeStop(...args);
       }}
     >
-      <NotebookContainer isResizing={isResizing}>
+      <Box w="100%" style={{ overflowY: getOverflow(isResizing) }}>
```

Comment:
I intentionally didn't re-create a component called `NotebookContainer` because we already have in at least one other place.